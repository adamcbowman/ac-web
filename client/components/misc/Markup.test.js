import React from 'react'
import Renderer from 'react-test-renderer'
import { Br, Credit, MultiLine } from './'

test('Br component', () => {
    expect(Renderer.create(<Br />)).toMatchSnapshot()
    expect(Renderer.create(<Br ribbon />)).toMatchSnapshot()
})

test('Credit component', () => {
    expect(Renderer.create(<Credit>Some credit</Credit>)).toMatchSnapshot()
    expect(
        Renderer.create(<Credit compact>Some credit</Credit>)
    ).toMatchSnapshot()
})

test('MultiLine component', () => {
    expect(
        Renderer.create(<MultiLine>Some content...</MultiLine>)
    ).toMatchSnapshot()
})
