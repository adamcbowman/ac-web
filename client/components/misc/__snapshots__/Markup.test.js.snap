// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Br component 1`] = `
<div
  className="Main"
/>
`;

exports[`Br component 2`] = `
<div
  className="Ribbon"
/>
`;

exports[`Credit component 1`] = `
<span
  className="Credit"
  data-label="Credit"
  onClick={[Function]}
>
  Some credit
</span>
`;

exports[`Credit component 2`] = `
<span
  className="Credit Compact"
  data-label="Credit"
  onClick={[Function]}
>
  Some credit
</span>
`;

exports[`MultiLine component 1`] = `
<p>
  Some content...
</p>
`;
